docs/_data/prague-2020-config.yaml

```diff
@@ -0,0 +1,138 @@
+name: Prague
+
+shortcode: prague # prague australia?
+year: 2020
+city: Prague
+local_area: Czech Republic
+area: Europe
+area_adj: European
+tz: CEST
+email: prague@writethedocs.org
+color: coral
```

Comment:
I don't know how to find the code for the 2020 color, @ericholscher halp?

Reply:
Ugh nevermind. It's just "blue" :)

Reply:
I thought the 2020 colors were the same for all conferences? Because this is copied from Portland 2020, which is coral.

Reply:
2020 config uses blue. Maybe coral in CSS and blue in config?